docs/source/unit.rst

```diff
@@ -108,8 +108,45 @@ Pipeline units of type :class:`Sieve <thoth.adviser.sieve.Sieve>` and
 <thoth.adviser.exceptions.NotAcceptable>`, see :ref:`sieves <sieves>` and
 :ref:`steps <steps>` sections for more info.
 
+Pipeline units of type :class:`Sieve` can also raise :class:`SkipPackage
+<thoth.adviser.exceptions.SkipPacakge>` to exclude the given package from an
+application stack completely. See :ref:`sieves <sieves>` section for more info.
+
 Raising any other exception in pipeline units causes resolver failure.
 
 All pipeline units should be atomic pieces and `they should do one thing and do
 it well <https://en.wikipedia.org/wiki/Unix_philosophy>`_. They were designed
 to be small pieces forming complex resolution system.
+
+Unit placement in a pipeline
+============================
+
+The pipeline configuration (which pipeline units in what configuration) is
+determined dynamically on each adviser start. This enables to construct the
```

Comment:
This enables construction of the pipeline depending on an input vector (...)

Reply:
Adjusted.

Reply:
Adjusted :+1: